jest.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   preset: 'react-native',
   setupFiles: ['./config/test/jest-setup.js'],
-  testRegex: '(/__tests__/.*|(\\.|/)(test|spec))(?<!\\.disabled)\\.[jt]sx?$',
-  testPathIgnorePatterns: ['node_modules', 'e2e'],
+  testMatch: ['**/*.(spec|test).[tj]s?(x)'],
+  testPathIgnorePatterns: ['node_modules', 'e2e', '\\.disabled\\.[jt]sx?$'],
```

Comment:
Let's us add helper utils inside of `__tests__`; same number of test files are run